Library/Homebrew/sorbet/rbi/gems/msgpack@1.7.2.rbi

```diff
@@ -1,319 +0,0 @@
-# typed: false
```

Comment:
This is unrelated to the scope of the PR, but Tapioca ends up setting this to `typed: false`, bc it conflicts with the source rbi [file](https://github.com/sorbet/sorbet/blob/master/rbi/gems/msgpack.rbi) in sorbet itself. We can just [exclude](https://github.com/Homebrew/brew/pull/16586/files#diff-e2fa03718f64a9a32fe9acef1ced320316bbb31161cbbaa74077c88cf24819feR52) it from generation (as we do with `json` for the same reason).